src/common/containers/container.css

```diff
@@ -2,11 +2,12 @@
   box-sizing: border-box;
   max-width: 940px;
   margin: 0 auto;
-  padding: 10px;
+  padding: 0 10px;
 }
 
 .container {
   composes: wrapper;
+  padding: 10px;
```

Comment:
Not sure if container needs top/bottom padding at all, but as other pages use it too I keep it around for backwards compatibility.